File: 04-switch-case/Calculator.java
import java.util.*;

public class Calculator {
    public static void main(String[] args) {
        System.out.println("Enter two number:");
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        System.out.println(
                "Enter the operation number you want to perfrom\n 1.Addition\n 2.Subtraction\n 3.Multiplication\n 4.Division\n 5.MOdulos or Remainder");
        int n = sc.nextInt();
        switch (n) {
            case 1:
                System.out.println(a + b);
                break;
            case 2:
                System.out.println(a - b);
                break;
            case 3:
                System.out.println(a * b);
                break;
            case 4: if(b==0){
                System.out.println("Invalid Division");
            } else {
                System.out.println(a / b);
            }
                break;
            case 5: if(b==0){
                System.out.println("Invalid Division");
            } else {
                System.out.println(a%b);
            }
                break;
            default:
                System.out.println("Enter the vaild number");
        }

    }
}

File: C++/Calculator.cpp
#include<iostream>
using namespace std;

int main(){

    int a;
    cout<<"Enter the value of first number:"<<endl;
    cin>>a;

    int b;
    cout<<"Enter the value of second number:"<<endl;
    cin>>b;

    int op;
    cout<<"Enter the operation number you want to perform--->";
    cout<<"Addition - 1"<<endl<<"Subtraction - 2"<<endl<<"Multiplication - 3"<<endl<<"Divison - 4"<<endl<<"Module - 5"<<endl;
    cin>>op;

    switch (op){
        case 1: cout<<"Addition: "<<a+b<<endl;
        break;

        case 2: cout<<"Subtraction: "<<a-b<<endl;
        break;

        case 3: cout<<"Multiplication: "<<a*b<<endl;
        break;

        case 4: cout<<"Division: "<<a/b<<endl;
        break;

        case 5: cout<<"Modules: "<<(a%b)<<endl;
        break;

        default: cout<<"invalid input";
    }
}
